uwblueprint/sistering
# User Account Creation Flow

## Ticket link

Closes #489, #399, #444, #440, #484, #469, #468 (this doesnt complete this task just yet, need to check 2 week time limit).



## Implementation description
* Integrated user invites in `/admin/users`, can now send user invites to emails
* users can click on link in email and go through entire account creation flow (create account page start, newaccountform)
* before user is created after form submission, we check if a userinvite exists (if so, create account, then proceed to account creation successful page), then the userinvite is deleted.



## Steps to test
1. Go to `/admin/users` and click on `Add Volunteer` and enter an email
2. Go through account creation process from email (it might be in spam)
3. once completed, try logging in to confirm it works.

Note: If you verify your email (you get another email after you create an account to verify your email with firebase), then you it will prevent you from using that same email to create a new account on sistering to test, so ignore the verification email. 


## Checklist
